sdk/communication/azure-communication-callautomation/src/main/java/com/azure/communication/callautomation/models/events/RecognizeCompleted.java

```diff
@@ -2,14 +2,16 @@
 // Licensed under the MIT License.
 // Code generated by Microsoft (R) AutoRest Code Generator.
 
-package com.azure.communication.callautomation.implementation.models;
+package com.azure.communication.callautomation.models.events;
 
+import com.azure.communication.callautomation.models.RecognitionType;
+import com.azure.communication.callautomation.models.CollectTonesResult;
 import com.azure.core.annotation.Fluent;
 import com.fasterxml.jackson.annotation.JsonProperty;
 
 /** The RecognizeCompleted model. */
 @Fluent
-public final class RecognizeCompleted {
+public final class RecognizeCompleted extends CallAutomationEventBase {
```

Comment:
I remember we were dropping the AutoGenerated events in Java. Maybe that's not the case anymore?